activity_browser/ui/widgets/biosphere_update.py

```diff
@@ -58,7 +47,8 @@ def run(self):
         try:
             for i, patch in enumerate(self.PATCHES):
                 self.progress.emit(i)
-                patch()
+                update_bio = getattr(UpdateBiosphereThread.data, patch)
+                update_bio()
         except ValidityError as e:
             print("Could not patch biosphere: {}".format(str(e)))
```

Comment:
Depending on when you plan to merge this, this should be `log.error` instead?

Reply:
this will be done later